astro/src/styles/bootstrap.scss

```diff
@@ -64,17 +64,34 @@ $custom-colors: (
   "ac": #041058,
   "ac2": #95B2E0,
   "ac3": #A9B6FB,
+  "light-secondary": #fcfeff,
```

Comment:
Let's see if instead of creating more themes, we learn to use the `bg-subtle` options that Bootstrap provides for the secondary and tertiary.

This would be done starting on line 100. If you want, let's jump on a call and walk through it.